invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/DocumentCirculation/DocumentCirculation.js

```diff
@@ -37,7 +37,8 @@ class BookAvailability extends Component {
     if (circulationData.has_items_for_loan > 0) {
       return (
         <List.Item>
-          <List.Icon name={'info'} />
+          <Popup content="Calculated based on current library resources"
```

Comment:
nit: `resources` -> `stock` or `availability`? It sounds money related to me :)

Reply:
Stock sounds the best to me.